### PR TITLE
CLC-5770 OEC: Implement MergeConfirmationSheetsTask

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/CLC-5770

Subtasks:
https://jira.ets.berkeley.edu/jira/browse/CLC-5771
https://jira.ets.berkeley.edu/jira/browse/CLC-5772

### CODE REVIEW

File: app/models/oec/export_task.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def run_internal
       students = Oec::Students.new
       course_students = Oec::CourseStudents.new
 
-      merged_supervisor_confirmations = @remote_drive.find_nested [@term_code, 'departments', 'merged_supervisor_confirmations']
+      merged_supervisor_confirmations = @remote_drive.find_nested [@term_code, 'departments', 'Merged supervior confirmations']
```
Comment (johncrossman) [inline app/models/oec/export_task.rb]: I like "supervior" but others might complain.

File: app/models/oec/merge_confirmation_sheets_task.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        course_confirmation_sheet = find_or_error('Courses', department_subfolder)
+        supervisor_confirmation_sheet = find_or_error('Report Viewers', department_subfolder)
+        sis_import_sheet = find_or_error(department_subfolder.title, most_recent_import)
+
+        course_confirmation = Oec::CourseConfirmation.from_csv @remote_drive.export_csv(course_confirmation_sheet)
+        supervisor_confirmation = Oec::SupervisorConfirmation.from_csv @remote_drive.export_csv(supervisor_confirmation_sheet)
+        sis_import = Oec::SisImportSheet.from_csv @remote_drive.export_csv(sis_import_sheet)
+
+        course_confirmation.each do |course_confirmation_row|
+          next unless course_confirmation_row['EVALUATE'] == 'Y'
```
Comment (johncrossman) [inline app/models/oec/merge_confirmation_sheets_task.rb]: Do we want case-insensitive compare on 'Y'?

**Comment by johncrossman** [comment]
This looks good to me, @paulkerschen  I'll merge once the above comments are resolved.

**Comment by paulkerschen** [comment]
The impervious supervior. Thanks, @johncrossman, comments addressed in amendment.

**Pull request merged**